server/src/routes/twiml.ts

```diff
@@ -1,12 +1,26 @@
 import { Request, RequestHandler, Response } from 'express';
-import { twiml } from 'twilio';
-import { TwilioCredentials } from '../common/types';
+import { twiml, validateRequest } from 'twilio';
+import { ServerCredentials } from '../common/types';
 
 export function createTwimlRoute(
-  twilioCredentials: TwilioCredentials,
+  serverConfig: ServerCredentials,
 ): RequestHandler {
   const { VoiceResponse } = twiml;
   return function twimlRoute(req: Request, res: Response) {
+    const twilioSignature = req.headers['x-twilio-signature'] as string;
+    const url = serverConfig.TWIML_REQUEST_URL;
+    const params = req.body;
+    const requestIsValid = validateRequest(
```

Comment:
I believe there's a function called `validateExpressRequest` or something along those lines in the `twilio` helper library that does all of this work for us, and I think it means we won't need to have `TWIML_REQUEST_URL` as an environment variable for us either, since it will pull the URL info out of the Express request. The latter I'm not too sure on, but the former is definitely worth checking out.